docs/GoogleDrive-batchsink.md

```diff
@@ -24,17 +24,23 @@ Is optional. In the case it is not set Google API will try to recognize file's M
 
 ### Authentication
 
-**Account file path:** Path on the local file system of the user/service account key used for authorization. 
+**Authentication type:** Defines the authentication type. OAuth2 and Service account types are available. 
```

Comment:
The description should answer the question "what it this is property?" not "what it does"

Reply:
done